assets/thanos-querier/deployment.yaml

```diff
@@ -61,6 +61,7 @@ spec:
         - --grpc-client-server-name=prometheus-grpc
         - --rule=dnssrv+_grpc._tcp.prometheus-operated.openshift-monitoring.svc.cluster.local
         - --target=dnssrv+_grpc._tcp.prometheus-operated.openshift-monitoring.svc.cluster.local
+        - --query.sd-dns-resolver=miekgdns
```

Comment:
```suggestion
        - --store.sd-dns-resolver=miekgdns
```

Reply:
4.13 branch https://github.com/openshift/cluster-monitoring-operator/blob/release-4.13/assets/thanos-querier/deployment.yaml#L56
is also
`- --store.sd-dns-resolver=miekgdns`